### DISCUSSION TITLE
Is it save to upgrade from 1.7RC4 to 1.7.14?

### DISCUSSION THREAD

**Discussion by SirezKN** [mozilla/rhino#1160] [Q&A] [answered]
Hi 👋 
is it save to upgrade from 1.7RC4 to 1.7.14?
Kind regards

**Comment by p-bakker** [comment, marked as answer]
Not sure exactly which version you're referring to when you say 1.7RC4, but in general Rhino tries hard to remain backwards compatible.

I'd check the release notes for all the intermediate versions and then just try out 1.7.14 with your codebase.

**Comment by SirezKN** [comment]
Thanks for the answer, we'll try :)
